scratch/app-github.jsonc

```diff
@@ -0,0 +1,39 @@
+{
+    "type": "view",
+    "bindings": [
+        {
+            "type": "menu",
+            "bindings": [
+                // Are these menu items meant to lead to rendering the list of open PRs with Mattermost, or is it a link to a page on GitHub with these listed?
+                {
+                    "type": "menu_item",
+                    "label": "Your Pull Requests",
+                    "icon": "icons/code-merge.png",
+                    "hint": "1",
+                    "submit": "views/your-pull-requests"
+                },
+                {
+                    "type": "menu_item",
+                    "label": "Pull Requests Needing Review",
+                    "icon": "icons/eye.png",
+                    "hint": "1",
+                    "submit": "views/your-review-requests"
+                },
+                {
+                    "type": "menu_item",
+                    "label": "Assignments",
+                    "icon": "icons/clipboard-person.png",
+                    "hint": "0",
+                    "submit": "views/your-assignments"
+                },
+                {
+                    "type": "menu_item",
+                    "label": "Unread Messages",
+                    "icon": "icons/envelope-closed.png",
+                    "hint": "0",
+                    "submit": "views/your-unread-messages"
+                }
+            ]
+        }
+    ]
```

Comment:
The screenshot associated has the bot. Is that the default behaviour? Or is also part of the definition of this screen?

Same for the introduction text. The design of this one seems slightly different from the design of the previous one.

Reply:
Design for the github RHS was a POC - the bot will likely be removed and just used in the LHS I think.

The intro/welcome block is intended to be a dismissable block for all app home screens